### PR TITLE
Bulk invite users form excel file feature

### PR DESCRIPTION
**Pull request by pahaz**
# Screens #

image
image
image

### CODE REVIEW

File: apps/_example05app/containers/FormTableExcelImport.js [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+            </Form.Item>
+        })}
+        <Form.Item>
+            <Button type="primary" htmlType="submit">{NextStepButtonLabelMsg}</Button>
+        </Form.Item>
+    </Form>
+}
+
+function ExcelExporterButton ({ columns, setExportedData }) {
+    if (!columns) throw new Error('no columns prop')
```
Comment (toplenboren) [inline apps/_example05app/containers/FormTableExcelImport.js:104]: Why not simply use [PropTypes](https://reactjs.org/docs/typechecking-with-proptypes.html) ? 

They allow us to explicitly specify the props, that are used in component and throw warnings if something is not right

File: apps/_example05app/utils/excal.utils.js [hunk 2]
```diff
@@ -0,0 +1,285 @@
+function isValidEmail (email) {
```
Comment (toplenboren) [inline apps/_example05app/utils/excal.utils.js:1]: File is called `excal`, why not `excel` ?

Comment (pahaz) [reply apps/_example05app/utils/excal.utils.js:1]: mistake %)

File: apps/_example05app/containers/FormTableExcelImport.js (hunk 1 above)
Comment (pahaz) [reply apps/_example05app/containers/FormTableExcelImport.js:104]: It's better to use PropTypes. I don't want to add PropTypes for some components it's better to add it to all public components. I think in the future we can refactor it. But regardless of this at the moment, we should avoid unclear errors for developers.

**Pull request merged**